tests/test_check.py

```diff
@@ -17,10 +17,9 @@ def test_clone_and_del_repo(git_path):
     # clone
     base_path = clone_repo(git_path)
     assert(base_path == os.path.basename(git_path))
-    # delete
-    deletion_status = del_repo(base_path)
-    if not(deletion_status): 
-        raise AssertionError
+
+    # delete should have return code of 0 (success)
+    assert del_repo(base_path) == 0
```

Comment:
Avoiding to use `assert()` is intentional because this is warned against in code quality checks.

Reply:
okay, just restored to previous way to raise the exception.